src/app/components/modules/Modals.jsx

```diff
@@ -30,6 +31,7 @@ class Modals extends React.Component {
         show_transfer_modal: false,
         show_confirm_modal: false,
         show_login_modal: false,
+        show_post_advanced_settings_modal: '',
```

Comment:
Why not bool?

Reply:
Ah I see because it is the formId that is saved here.

Reply:
Ah yes, that's odd, I remember putting in a // formId comment there, but maybe it's somewhere else